### PR TITLE
Fixes some issues

### PR DESCRIPTION
**Pull request by Ekultek**
- Have to have the right version to create an issue
 - Changes the way that issue "identifiers" are created
 - Some minor fixes

### CODE REVIEW

**Comment by Ekultek** [comment]
Also the docker stuff

**Comment by NullArray** [comment]
Thought i merged the docker stuff already? Anyway i'll add my review.

File: Docker/README.md [hunk 1]
```diff
@@ -1,75 +1,19 @@
 # Docker deployment instructions
```
Comment (NullArray) [inline Docker/README.md]: Why are we getting rid of this?

**Review by NullArray — CHANGES_REQUESTED** [review]
Why are we getting rid of the README.md in the docker directory. I have referenced that in the main README, with regards to the docker compose stuff.

File: Docker/README.md (hunk 1 above)
Comment (Ekultek) [reply Docker/README.md]: Oops didn’t pull before pushing! One second

**Comment by NullArray** [comment]
If i am missing something, please let me know. It could be an oversight on my part but i'd like to make sure before i merge it into the master branch.

File: Docker/README.md (hunk 1 above)
Comment (NullArray) [reply Docker/README.md]: No problem <3

**Review by NullArray — APPROVED** [review]
Thank you for addressing the issue. Approved and merged.

**Comment by Ekultek** [comment]
<3

**Comment by kuswahaakhilesh51-png** [comment]
Hi

**Pull request merged**